dev/com.ibm.ws.jdbc_fat_derby/test-applications/jdbcapp/src/jdbc/fat/derby/web/JDBCDerbyServlet.java

```diff
@@ -46,54 +87,72 @@
     @Resource(name = "jdbc/dsfat3ref", lookup = "jdbc/dsfat3")
     DataSource ds3; //DSConfig (TRAN_NONE) + Res-ref (TRAN_NONE) + TRAN_NONE JDBC Driver
 
+    @Resource(name = "jdbc/dsfat5")
+    DataSource ds5; //DSConfig (TRAN_NONE) + No Res-ref + Normal JDBC Driver
```

Comment:
For those of these that are only used in one test and unlikely to be useful for other tests can we move the lookup to the test case to reduce clutter.